lua/review.lua

```diff
@@ -1,6 +1,6 @@
 -- -*- coding: utf-8 -*-
 -- Re:VIEW Writer for Pandoc
--- Copyright 2020-2023 atusy and Kenshi Muto
+-- Copyright 2020 Kenshi Muto
```

Comment:
```suggestion
-- Copyright 2020-2023 atusy and Kenshi Muto
```
変えたのがrevertしちゃったっぽいので戻してみます。copyright、ほかに使っているお名前があればそれを使っていただいても!

Reply:
atusyしか使ってないので、これで。ありがとうございます。